crates/sui-indexer-alt/src/models/transactions.rs

```diff
@@ -46,3 +49,35 @@ pub struct StoredTxBalanceChange {
     pub tx_sequence_number: i64,
     pub balance_changes: Vec<u8>,
 }
+
+#[derive(Insertable, Debug, Clone)]
```

Comment:
nit: Alphabetic order here as well please (In this case, I think you just need to push `StoredTxAffectedAddress` up by a couple of slots).